sermant-backend/src/main/webapp/frontend/src/views/ConfigView.vue

```diff
@@ -274,6 +274,19 @@ const fillServiceNames = () => {
   });
 }
 
+const resultCodeMap = new Map<string, string>([
+  ['00', '成功'],
+  ['01', '无法连接配置中心'],
+  ['02', '配置查询失败'],
+  ['03', '配置已存在'],
+  ['04', '新增配置失败'],
+  ['05', '发布配置失败'],
+  ['06', '删除配置失败'],
+  ['07', '配置不存在'],
+  ['08', '缺少请求参数'],
+  ['09', '请求失败'],
+]);
+
```

Comment:
extract this to common place for multiple usage

Reply:
done